### ISSUE TITLE
no windows transparent why?

### ISSUE THREAD

**Issue by crashangelbr** [LukeSmithxyz/voidrice#8]
Hi, i install void linux and extract yous files... when i startx the windows not transparent, why? top panel not visible.. why?

thanks...

**Comment by LukeSmithxyz** [comment]
You're probably lacking the required dependencies. `polybar` must be installed for the status bar and transparency requires `compton`.

**Comment by Panadestein** [comment]
Same issue over here, `compton` already installed, and also `polybar`.

**Comment by LukeSmithxyz** [comment]
Yes, that's intentional in the sense that now that I don't really use transparency, I don't really use compton, so it doesn't autostart in my i3 config. You'll notice there's a commented out line to start it in the i3 config.

On the font resize issue, do you have `urxvt-resize-font-git` installed?

**Comment by MJGrey** [comment]
Hey Luke, I seem to be having a similar issue, I found the compton line that was commented out but the 2nd part of my problem is with urxvt, its still got the white background and I can't seem to get rid of it. Am I missing something in your .Xdefaults file by any chance? (its a fresh install on Ubuntu 17.04 if that counts for anything, still working up the courage to build my Arch distro.)

**Comment by Panadestein** [comment]
I fixed the problem. It was just a matter of edit the file `~/.Xdefaults` and add the corresponding entries for for  urxvt transparency (see for instance [)](https://bbs.archlinux.org/viewtopic.php?id=117543). With respect to the font size, what I meant is that it is not longer customable from `~/.config/i3/config` but using   `~/.Xdefaults` instead. 

Additionally (and I am sorry, this should be another thread) there is a conflict between two alias in the generated `~/.bashrc`, for instance `v` makes reference to `alias v="cd ~/Videos"` and to `alias v="vim"`, which I solved changing one of them.

Right now I am actively testing the rice created using the amazing [LARBS](http://larbs.xyz/) in a fresh Arch Linux install, as can be noticed. Things that I would like to fix in addition to the previous issues: 

- Lagging in urxvt when using `vim` with large files 

- Lost the colors of the PS1 when in tty. 

- Some times executing `$mod+Shift+Esc` log me out but enters in a loop of searching email address. Probably related with mutt config?

And that's it by now, thanks again for the wonderful job Luke. I will keep using it and having fun!
